assets/ui.css

```
@@ -426,3 +426,22 @@
 #xref-ui:not(.ready) {
   background: url("https://respec.org/xref/loader.gif") no-repeat center;
 }
+
+.respec-dfn-list li span.exported, span.unused {
+  margin-left: 8px;
```

Nit: for consistency with rest of styles.
```suggestion
  margin-left: 0.5em;
```